examples/camera_streamer/camera_config.py

```
@@ -0,0 +1,180 @@
+        "type",
+        "stereo",
+        "width",
+        "height",
+        "fps",
+        "streams",
+        "enabled",
+        "serial_number",
+        "resolution",
+        "device_id",
+        "device",
+    }
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)